docs/pipelines.md

```diff
@@ -489,7 +489,11 @@ they are referenced in the `Pipeline` definition.
 For each `Task` in the `Pipeline`, you can specify the number of times Tekton
 should retry its execution when it fails. When a `Task` fails, the corresponding
 `TaskRun` sets its `Succeeded` `Condition` to `False`. The `retries` field
-instructs Tekton to retry executing the `Task` when this happens.
+instructs Tekton to retry executing the `Task` when this happens. `retries` are executed
+even when other `Task`s in the `Pipeline` have failed, unless the `PipelineRun` has
+been [cancelled](./pipelineruns.md#cancelling-a-pipelinerun),
+[gracefully cancelled](./pipelineruns.md#gracefully-cancelling-a-pipelinerun), or
+[gracefully stopped](./pipelineruns.md#gracefully-stopping-a-pipelinerun).
```

Comment:
didn't realize we're explicit about not exhausting retries in cancelation and graceful termination cases, but I see we've documented it in the [docs](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#cancelling-a-pipelinerun), now this makes me want for us to be consistent by not exhausting retries across the board instead of having exceptions 🤔 

(apologies for revisiting the discussion about what to do, no one felt strongly in the API WG meeting so hopefully could be an easy switch 🤞🏾)